SageOfCode/Crate
# Feature set up

#### What does  this PR do?
- Set up the project with required dependencies and installs
- Connect the Store to Redux Dev Tools
- Add a Pull Request template

#### Where should the reviewer start?
- package-lock.json files
- package.json files
- code/web/src/setup/store.js

#### How should this be manually tested?
- Pull down branch and run "npm start" in code/web and code/api to ensure that the app is running
- Open the dev tools and look into the Redux section

#### Any background context you want to provide?
- This PR is primarily just the set up of the project, but it does add dev tools for redux and a PR template

#### What are the relevant tickets?
- N/A
